market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java

```diff
@@ -23,9 +23,31 @@ public class MarketMetaSchema {
   public String version;
   @NotNull @Size(min = 4, max = 24)
   public String name;
+  public static class DisplayName {
+    @NotNull
+    @Size(min = 2)
+    public String locale;
```

Comment:
I'd give examples here; `@Examples({'de', 'en'})` so if one edits these product descriptions with a real editor there will be auto-completion support.

Reply:
I added example for it.